.circleci/config.yml

```diff
@@ -0,0 +1,67 @@
+# Golang CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2
+jobs:
+  build:
+    machine: true
+    # docker:
+    #   # specify the version
+    #   - image: circleci/golang:1.11
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/postgres:9.4
+
+    #### TEMPLATE_NOTE: go expects specific checkout path representing url
+    #### expecting it in the form of
+    ####   /go/src/github.com/circleci/go-tool
+    ####   /go/src/bitbucket.org/circleci/go-tool
+    working_directory: /home/circleci/go/src/github.com/src-d/gitbase
+    steps:
+      #- setup_remote_docker
+      - checkout
+      - run: &install
+          name: Install dependancies 
+          command: |
+            sudo apt-get update
+            sudo apt-get install build-essential software-properties-common -y
+            sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+            sudo apt-get update
+            sudo apt-get install -y gcc g++-6
+            wget https://dl.google.com/go/go1.11.linux-amd64.tar.gz
```

Comment:
do we need all this boilerplate to be able to build and test go apps on circleci?

Reply:
Unfortunately, yes (see next comment)